src/modules/events/send_event.h

```diff
@@ -51,46 +51,93 @@ extern "C" __EXPORT int send_event_main(int argc, char *argv[]);
 class SendEvent : public ModuleBase<SendEvent>, public ModuleParams
 {
 public:
+
+	/**
+	 * @brief Default Constructor.
```

Comment:
Thanks @bkueng , I understand where you are coming from as it pertains to this change.  I am trying to help craft a baseline where documentation (and documentation updating) is a normal part of the workflow.  The reason this is important to me is not for the seasoned veterans, but rather for the newcomers.  If that doesn't really fit for this file, I understand and I'll back it out shortly.  Thank you for your feedback!

-Mark

Reply:
Hi @bkueng, I've backed out the constructor/destructor comments and tested all functionality on pixracer.  I know we've discussed previously, but I can't see anything using send_event outside of temperature calibration.  Does send_event get used by anything else that I should check?

Thanks!

-Mark

Reply:
>  The reason this is important to me is not for the seasoned veterans, but rather for the newcomers

That's fine with me - I regularily write documentation (e.g. dev-guide) and I don't do it for seasoned veterans. In this particular case it sill does not help, because if someone does not know what a default constructor is, he/she simply cannot reasonably contribute/work with the code.

>  I know we've discussed previously, but I can't see anything using send_event outside of temperature calibration. Does send_event get used by anything else that I should check?

It's used for RC loss alarm for instance: https://github.com/mcsauder/Firmware/blob/2fa7e9310d31d4c10f15a292cee8c51492e84b21/src/modules/events/rc_loss_alarm.cpp.